beacon-chain/rpc/prysm/v1alpha1/validator/proposer_execution_payload.go

```diff
@@ -127,8 +127,8 @@ func (vs *Server) getLocalPayload(ctx context.Context, blk interfaces.ReadOnlyBe
 	}
 	var attr payloadattribute.Attributer
 	switch st.Version() {
-	case version.Deneb:
-		withdrawals, err := st.ExpectedWithdrawals()
+	case version.Deneb, version.Electra:
```

Comment:
Same comment as above: in some places, we add the Electra version, but in some places, we do not.

Reply:
This PR doesn't address all of the places that need to handle electra cases in switch statement. In fact, this change is probably out of scope of this PR. I only added it since I was here due to API changes on `st.ExpectedWithdrawals()`

Reply:
Agree. I think we should either do it all or not do it at all. Otherwise, the disparity might be confusing and lead to missing test coverage down the line.

Reply:
OK, I'll revert it here

Reply:
Reverted in d8ab9dff976bd93297629dc21c55d3d27af13de8